support-files/mariadb.service.in

```diff
@@ -84,7 +83,7 @@ ExecStart=@sbindir@/mysqld $MYSQLD_OPTS $_WSREP_NEW_CLUSTER $_WSREP_START_POSITI
 @SYSTEMD_EXECSTARTPOST@
 
 # Unset _WSREP_START_POSITION environment variable.
-ExecStartPost=/bin/sh -c "systemctl unset-environment _WSREP_START_POSITION"
+ExecStartPost=@bindir@/rm @MYSQL_UNIX_DIR@/wsrep-start-position
```

Comment:
`@bindir@` is referring to the mariadb location, so I think an absolute `/bin/rm` is safe here, especially as systemd is linux only.

Reply:
I was not certain about what path to choose. Can we be sure `/bin/rm` will not break?
(For Arch it is fine.)

Reply:
Just a few tests:
<pre>
# docker run ubuntu:18.04 ls -la /bin/rm /usr/bin/rm
ls: cannot access '/usr/bin/rm': No such file or directory
-rwxr-xr-x. 1 root root 63704 Jan 18  2018 /bin/rm

# docker run ubuntu:16.04 ls -la /bin/rm /usr/bin/rm
ls: cannot access '/usr/bin/rm': No such file or directory
-rwxr-xr-x. 1 root root 60272 Mar  2  2017 /bin/rm

# docker run ubuntu:14.04 ls -la /bin/rm /usr/bin/rm
ls: cannot access /usr/bin/rm: No such file or directory
-rwxr-xr-x. 1 root root 60160 Mar 10  2016 /bin/rm

# docker run alpine ls -la /bin/rm /usr/bin/rm
ls: /usr/bin/rm: No such file or directory
lrwxrwxrwx    1 root     root            12 Jan  9  2018 /bin/rm -> /bin/busybox

# docker run debian:stable ls -la /bin/rm /usr/bin/rm
ls: cannot access '/usr/bin/rm': No such file or directory
-rwxr-xr-x. 1 root root 64424 Feb 22  2017 /bin/rm

# docker run debian:oldstable ls -la /bin/rm /usr/bin/rm
ls: cannot access /usr/bin/rm: No such file or directory
-rwxr-xr-x. 1 root root 60072 Mar 14  2015 /bin/rm

# docker run centos:6 ls -la /bin/rm /usr/bin/rm
ls: cannot access /usr/bin/rm: No such file or directory
-rwxr-xr-x. 1 root root 53592 Mar 22  2017 /bin/rm

# docker run centos:7 ls -la /bin/rm /usr/bin/rm
-rwxr-xr-x. 1 root root 62864 Nov  5  2016 /bin/rm
-rwxr-xr-x. 1 root root 62864 Nov  5  2016 /usr/bin/rm

# docker run fedora:29 ls -la /bin/rm /usr/bin/rm
-rwxr-xr-x. 1 root root 86432 Nov  7 15:14 /bin/rm
-rwxr-xr-x. 1 root root 86432 Nov  7 15:14 /usr/bin/rm
</pre>